app/controllers/verify/review_controller.rb

```diff
@@ -23,6 +23,14 @@ def confirm_current_password
     def new
       idv_session.params.symbolize_keys!
       analytics.track_event(Analytics::IDV_REVIEW_VISIT)
+
+      phone_of_record_msg = ActionController::Base.helpers.content_tag(
+        :strong, t('idv.messages.phone.phone_of_record')
+      )
+
+      # TODO: why doesn't this correctly output html without `html_safe`?
```

Comment:
Yeah that's what I was saying, it's a huge change and year it may be out of scope for this PR but I think it might be worthwhile considering as an alternative

But re: `phone_message` -- it looks like all the flash messages we had are static strings, and don't have any dynamic interpolation, so I think we could handle it all with a single key (the phone message could be rewritten with a `<strong>` in the key (I know I usually disagree with this) but it's an idea?

Anyways, I think `sanitize` might be temporarily OK but I think we can try to build safer ways to transport HTML strings. Or maybe even considering removing the formatting entirely?